### ISSUE TITLE
Add Code Preview to Import Data "From Text (base)" or consolidate dialog with other dialogs

### ISSUE THREAD

**Issue by BdR76** [rstudio/rstudio#13609]
In the "Import Dataset" on the environment tab (top right) there are several options to import a dataset from a file. All options have a "Code Preview" where you can copy the generated code, except for "From Text (base)".

![rstudio_import_dataset](https://github.com/rstudio/rstudio/assets/7793956/f0c331f0-f0c9-49cf-b9ca-3aea3dab7774)

The first option "From Text (base)" is the odd one out, the dialog looks different than the others and it doesn't have a "Code Preview" textbox.

![rstudio_import_text_base](https://github.com/rstudio/rstudio/assets/7793956/44d32131-ed1b-42fe-b9f6-0847aa7a6336)

![rstudio_import_text_readr](https://github.com/rstudio/rstudio/assets/7793956/cf3973fd-6e0d-4bcd-ab19-79b1eb8c3471)

Importing a dataset through the GUI is very helpful to select the correct import options for a given file and get a live-preview of the result. Most of the time you want to generate it as a script so that it is repeatable for other data managers. So imho it's odd that the Code Preview is missing in that one dialog.

Is it possible to either

1) add "Code Preview" texbox (+copy button) also to the first "From Text (base)" dialog?

or 

2) Make the "From Text (base)" dialog the same as the other 5 dialogs, which all have a "Code Preview" part.

**Comment by ronblum** [comment]
@BdR76 Good point! I'll mark this for review as we continue development of RStudio.

**Comment by BdR76** [comment]
@ronblum I got an automatic notice that this issue is "stale", are there any updates?

**Comment by BdR76** [comment]
Just answering to remove the "stale" label. Btw I didn't notice before but the code is just one line and it is already shown in the console, which you can copy from there. So after clicking the `Import` button this will for example appear in the console:

```
> mydata123 <- read.csv("~/mydata123.csv", header=TRUE, sep=";")
> View(mydata123)
```
